cypress.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Are you planning on adding a CI process to run these cypress tests during PRs? Or only manually? I am new to Cypress and might be missing something, but think the greatest value would be if we use Travis CI, Github Actions, or similar to run the tests for each branch / PR.

I understand that getting CI working can also be a challenge, so if you'd like to wait on that for a future PR, that's okay with me, too.

Reply:
> Are you planning on adding a CI process

For sure, captain! :slightly_smiling_face: